### PR TITLE
Convert gas values to integers

### LINKED ISSUE #370: gasPrice RPC gives back number instead of HEX
The docs state it should return a HEX, though it returns a number as string:
https://github.com/ethereum/wiki/wiki/JSON-RPC#eth_gasprice

### PR DESCRIPTION
**Pull request by tgerring**
Fixes #370

### CODE REVIEW

**Comment by frozeman** [comment]
I tested your branch and the output is not correct. It returns a HEX with two leading `0x`:

```js
{"id":65,"jsonrpc":"2.0","result":"0x0x09184e72a000"}
```

**Comment by tgerring** [comment]
@frozeman duplicate "0x" should be removed now, thanks for catching that!

File: rpc/packages.go [hunk 1]
```diff
@@ -45,17 +40,22 @@ type EthereumApi struct {
 	register map[string][]*NewTxArgs
 
 	db ethutil.Database
+
+	defaultGasPrice *big.Int
```
Comment (obscuren) [inline rpc/packages.go]: These should just be global variables

**Comment by obscuren** [comment]
:+1:

**Pull request merged**